src/views/results/components/ResultsList.tsx

```diff
@@ -39,9 +56,20 @@ export const ResultsList: FunctionComponent<Props> = (
             </button>
           );
         })}
+        <br />
```

Comment:
Ei kannata käyttää HTML-elementtejä sivun välyksien säätämiseen. CSS-tyylitiedostot sopii paremmin tähän tarkoitukseen. Jos tarvii väliä, niin määrittele jollekin ympärillä olevalle elementille `margin`ia.

Projekti käyttää tällä hetkellä CSS-tiedostoja, mitä yritetään pitää järjestyksessä nimeämiskäytännöillä. Esimerkiksi Results-näkymän tyylit on tiedostossa https://github.com/Archinowsk/konsti-client/blob/master/src/styles/results.css. CSS-tiedostoista viitataan HTML-elementteihin `className` parametrilla, esim näin: `<div className='results-list'>`.

Haluaisin vaihtaa irralliset CSS-tiedostot tyylitettyihin komponentteihin (`styled-components`), jolloin komponentti ja tyylit ovat samassa tiedostossa, mutta tätä ei ole vielä tehty.